services/common/src/constants/fileTypes.js

```diff
@@ -30,6 +30,11 @@ export const IMAGE = {
   ".jpg": "image/jpg",
   ".jpeg": "image/jpeg",
   ".png": "image/png",
+  ".jp2": "image/jp2",
+  ".tif": "image/tiff",
+  ".tiff": "image/tiff",
+  ".heic": "image/heic",
+  ".heif": "image/heif",
 };
 
```

Comment:
I think we'll want to create the message file type here as well to include in this update.  

Something like this:
```suggestion
export const MESSAGE = {
  ".msg": "application/vnd.ms-outlook",
};
```

Reply:
Thanks Done!